src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -134,6 +175,7 @@ export const UnboundDiskNotification = props => {
       padding: '1rem',
       border: '1px solid',
       borderColor: colors.accent(),
+      marginTop: '1rem',
```

Comment:
I am assuming that any workspace showing this notification will also have the new warning, so adding space between them.

![image](https://user-images.githubusercontent.com/484484/202548340-2d0ce522-c69c-48ab-b4f6-421523e7f145.png)

Reply:
I assume these warnings are both going away after the specified dates? 
If this were something longer term, I would say the css seems finicky and fragile for that, and getting flex to handle the spacing would be a good idea, but for something that is going to get pulled out in a couple of months, the quick and dirty solution makes more sense.
This was a very long-winded way of basically nodding along while reading....